docs/create-an-account.md

```diff
@@ -1,61 +1,33 @@
 # Create an account
 
 In order to request St. Jude data and work with that data in the cloud, you will need to create a DNAnexus account and use it to log in to St. Jude Cloud. St. Jude Cloud is built on top of the [DNAnexus](https://www.dnanexus.com/) genomics cloud ecosystem. For a comprehensive overview of how DNAnexus works, see 
-[this page](https://www.dnanexus.com/product-overview). Each new user whether regular or internal will receive a $50 credit upon creation of their DNAnexus account.
+[this page](https://www.dnanexus.com/product-overview). Each new user will receive a $50 credit upon creation of their St. Jude Cloud account with DNAnexus.
```

Comment:
They don't receive any credit until they add their billing info (which does **not** require credit card information) under their DNAnexus profile. I think we should mention that and link to the `Billing Setup` section.